packages/@aws-cdk/alexa-ask/package.json

```diff
@@ -20,7 +20,11 @@
           "artifactId": "alexa-ask"
         }
       },
-      "sphinx": {}
+      "sphinx": {},
+      "awsdocs": {
+        "title": "Alexa Ask",
+        "tags": ["alexa"]
```

Comment:
Where do the tags surface? I wonder if we could re-use the package's keywords for that (which might mean tweak them somewhat)

Reply:
Not anywhere yet, but in my mind the landing page of the AWS Construct Library will list the tags and the packages that apply to them.